db/migrate/20170110090935_add_ems_to_miq_alert_status.rb

```diff
@@ -0,0 +1,31 @@
+class AddEmsToMiqAlertStatus < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    has_many :vms, :class_name => "AddEmsToMiqAlertStatus::Vm"
+  end
+
+  class Vm < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    belongs_to :ext_management_system, :class_name  => "AddEmsToMiqAlertStatus::ExtManagementSystem",
+                                       :foreign_key => :ems_id
+    has_many :miq_alert_statuses, :class_name => "AddEmsToMiqAlertStatus::MiqAlertStatus", :as => :resource
+  end
+
+  class MiqAlertStatus < ActiveRecord::Base
+    belongs_to :resource, :polymorphic => true
+  end
+
+  def up
+    add_column :miq_alert_statuses, :ems_id, :bigint
+    say_with_time("add ems_id to miq alert statuses") do
+      MiqAlertStatus.all.each do |mas|
```

Comment:
darn - also of node, `:ems_id => Arel.sql("(select 1)")` converts to `ems_id = 0`